webapp/channels/src/components/integrations/outgoing_oauth_connections/installed_outgoing_oauth_connection.tsx

```diff
@@ -123,13 +121,10 @@ const InstalledOutgoingOAuthConnection = (props: InstalledOutgoingOAuthConnectio
             <div className='item-details__row'>
                 <span className='item-details__token'>
                     <FormattedMessage
-                        id='installedIntegrations.clientId'
-                        defaultMessage='Client ID: <b>{clientId}</b>'
-                        values={{
-                            clientId: outgoingOAuthConnection.client_id,
-                            b: (chunks: string) => <b>{chunks}</b>,
-                        }}
+                        id='installed_integrations.client_id'
+                        defaultMessage='Client ID: '
                     />
+                    <strong>{outgoingOAuthConnection.client_id}</strong>
```

Comment:
Just wondering why change this one. There's sometimes reason to change the way ordering works in other languages. Maybe not this particular one - but just wondering why.

Reply:
It's because the translated text is a label for the data as opposed to other cases where the text is a sentence or phrase where order matters more. The only time we might care about the order of the label is possibly with a RTL language, but we don't really support those, and I think that might normally be handled by CSS or some browser setting.

In general, there's a bit of performance overhead for using the rich formatting features as well as being a bit more work for translators. I don't think those benefits are big enough to justify changing this everywhere because any change makes more work for translators, but since they already have to update the translation to remove the `FormattedMarkdownMessage`, I figured I may as well change this at the same time